### PR TITLE
Broken Link Update

### PR DESCRIPTION
**Pull request by LalithManokar**
Link Update

### CODE REVIEW

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please update the tags. The name space "products" has been deprecated over a year ago.
Please check https://imsproddd27584c4.us2.hana.ondemand.com/ui/tags for the valid tags.

**Review by LalithManokar — COMMENTED** [review]
Done Changes

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 parser: v2
 auto_validation: true
 time: 45
-tags: [ tutorial>intermediate, products>products>sap-btp--cloud-foundry-environment\, kyma-runtime]
-primary_tag: products>sap-btp--cloud-foundry-environment
+tags: [ tutorial>intermediate, software-product>sap-business-technology-platform\, kyma-runtime]
```
Comment (MichaelCzcz) [inline tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md:5]: Please remove backslash and correct tag for kyma runtime (software-product>sap-btp--kyma-runtime)

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md (hunk 1 above)
Comment (LalithManokar) [reply tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md]: @MichaelCzcz done

**Pull request merged**